plugin/othttp/config.go

```diff
@@ -0,0 +1,140 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package othttp
+
+import (
+	"net/http"
+
+	"go.opentelemetry.io/otel/api/propagation"
+	"go.opentelemetry.io/otel/api/trace"
+)
+
+// Config represents the configuration options available for the othttp.Handler
+// and othttp.Transport types.
+type Config struct {
+	Tracer            trace.Tracer
+	Props             propagation.Propagators
```

Comment:
I suggest spelling this out "Propagators", otherwise it reads like a shortening of "Properties".

Reply:
Yup, I had the same thought myself but stuck with what the handler had called it.  Since I'm not the only one who thinks that I'll spell it out.